bin/gofmt.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nash
+
+import "io"
+import "acme/acme.sh"
+
+fn gofmt(file) {
```

Comment:
Really nice :D, doesn't ACME already comes with some stuff to use with Go ? (like formatting) ?

Reply:
no.. this go against the very idea of acme of being universal and language independent. 
but there's the scripts/programs of renowned acme users like pike, russ, dmr, etc, online. Most of them are `rc` scripts, but russ uses some Go programs.